apps/rush-lib/src/logic/policy/LockfilePolicy.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
Should this be called `PnpmShrinkwrapPolicy`?  It seems to be PNPM-specific.

Reply:
There's no reason it couldn't be implemented for other package managers but yes for now it is PNPM-specific. I'll rename to ShrinkwrapFilePolicy to keep consistency with other shrinkwrap stuff. Let me know if you still think `Pnpm` needs to be specified